### DISCUSSION TITLE
Allow to preserve ratio while resize width and height of the selection in the beatmap editor at the same time while dragging selection corner.

### DISCUSSION THREAD

**Discussion by Trung0246** [ppy/osu#22856] [Ideas]
Currently with the current resizing mechanism it is impossible to preserve distance between notes as user have to either manually have to resize only width or height at once. This feature allows perfectly preserve ratio while resize a selected note group and preserve distance between notes. Maybe something like holding shift while dragging corner?

**Comment by peppy** [comment]
Definitely sounds required.

Maybe even the default, and hold shift to disable?
